docs/reference/commandline/dockerd.md

```diff
@@ -828,6 +829,32 @@ To set the DNS search domain for all Docker containers, use:
 $ sudo dockerd --dns-search example.com
 ```
 
+#### Allow push of nondistributable artifacts
+
+Some images (e.g., Windows base images) contain artifacts whose distribution is
+restricted by license. When these images are pushed to a registry, restricted
+artifacts are not included.
+
+To override this behavior for specific registries, use the
+`--allow-nondistributable-artifacts` option in one of the following forms:
+
+* `--allow-nondistributable-artifacts myregistry:5000` tells the Docker daemon
+  to push nondistributable artifacts to myregistry:5000.
+* `--allow-nondistributable-artifacts 10.1.0.0/16` tells the Docker daemon to
+  push nondistributable artifacts to all registries whose resolved IP address is
+  within the subnet described by the CIDR syntax.
+
+This option can be used multiple times.
+
+This option is useful when pushing images containing nondistributable artifacts
+to a registry on an air-gapped network so hosts on that network can pull the
+images without connecting to another server.
+
+**Warning**: Nondistributable artifacts typically have restrictions on how and
+where they can be distributed and shared. Only use this feature to push
+artifacts to private registries and ensure that you are in compliance with any
+terms that cover redistributing nondistributable artifacts.
```

Comment:
Can you format the warning block as a markdown `quote` here? (put a `>` + `<space>` at the start of each line) that way it gets properly formatted in the online docs